php/util/class-fieldmanager-util-validation.php

```diff
@@ -356,8 +357,8 @@ private function format_value( $value ) {
 	 * @return string The field name with quotes added if necessary
 	 */
 	private function quote_field_name( $field ) {
-		// Check if the field name is alphanumeric (underscores and dashes are allowed).
-		if ( ctype_alnum( str_replace( array( '_', '-' ), '', $field ) ) ) {
+		// Check if the field name is alphanumeric (underscores are allowed).
+		if ( ctype_alnum( str_replace( array( '_' ), '', $field ) ) ) {
```

Comment:
Since the search is no longer multiple strings, it should just be a string vs an array